src/python/turicreate/toolkits/_model.py

```diff
@@ -115,7 +115,17 @@ def load_model(location):
                     model = _extensions.style_transfer()
                     model.import_from_custom_model(model_data, model_version)
                     return cls(model)
-                    
+
+                if name=='one_shot_object_detector' and OD_USE_CPP:
+                    od_cls = MODEL_NAME_MAP['object_detector']
+                    if 'detector_model' in model_data['detector']:
+                        model_data['detector'] = od_cls(model_data['detector']['detector_model'])
+                    else:
+                        model = _extensions.object_detector()
```

Comment:
We should always be using extensions. We should never use the old Python class. We are going to delete it.

Reply:
Oh it is not a python model,
`if 'detector_model' in model_data['detector']` means the od model is saved as native model,
`else` means that the ob model is saved as version 5.8 or older.
But `od_cls` is always an `Object_detector_beta` object.